testing/helpers/test-helpers.bash

```diff
@@ -30,3 +34,101 @@ function ensure_variables_set() {
         exit 1
     fi
 }
+
+# Converts the value of $COUCHBASE_SERVER_HOSTS into a file ready to feed to Prometheus file_sd.
+# Stores the name of the generated file in $PROMETHEUS_TARGETS_FILE.
+function _create_prometheus_targets_file() {
+    local tmp
+    tmp=$(mktemp -d)
```

Comment:
Why directory and not just a temp file for the smoke config? Ah for the .json suffix?

What about the BATS temporary locations?